public/mobile-app/src/routes/+page.svelte

```diff
@@ -92,7 +93,7 @@ function dismissError() {
 </script>
 
 <div class="homepage">
-{#if !isFranceConnected}
+{#if isFranceConnected === false}
```

Comment:
This looks like a code smell: using a "three value boolean" 😬 

What happens if `isFranceConnected` is null? Should we instead check if it's NOT france connected, as previously?

Reply:
isFranceConnected is null while waiting for the response to checkAuth: in this case, I prefer to have a blank page rather than a FranceConnexion button, while waiting for a skeleton.